docs/services/vision/mlmodel.md

```diff
@@ -223,15 +223,24 @@ If the classifier's results exceed the confidence threshold, the **Run model** s
 
 ### Live camera footage
 
-If you intend to use the detector or classifier with a camera that is part of your machine, you can test your detector or classifier from the [**Control tab**](/fleet/control/) or with code:
+You can test your detector or classifier from the [**Control tab**](/fleet/control/) or with code using a camera that is part of your machine.
 
-1. Configure a [camera component](/components/camera/).
+#### Test with the CONTROL tab
 
+1. Configure a [camera component](/components/camera/).
    {{< alert title="Tip" color="tip" >}}
    This is the camera whose name you need to pass to vision service methods.
    {{< /alert >}}
 
-2. (Optional) If you would like to see detections or classifications from the **Control tab**, configure a [transform camera](/components/camera/transform/) with the following attributes:
+2. After adding the camera, click the **Save** button in the top right corner of the page.
+3. Click on the test tab or navigate to the **CONTROL** tab, click on the vision service and select your camera and vision service and then click **Refresh**.
+   The panel will show detections with bounding boxes around detections on the image.
+
+![Blue boxes detected](/services/vision/detections.png)
+
+{{% expand "Click to see how to configure a camera live feed that shows detections or classifications" %}}
```

Comment:
question-- do you know if this is getting phased out and if so might it be best to just recommend vision control card and explain more about refreshing to fetch detections?

Reply:
It's not getting phased out. It's just no longer the only way to see the detections in the UI. Once the test card has classifications we could remove this though